<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<<<<<<< HEAD
<testsuite skipped="0" hostname="DESKTOP-A6N9SAN" name="testScripts.Motorcycle.EnterInsuranceData" tests="25" failures="0" timestamp="21 Aug 2020 10:28:15 GMT" time="64.900" errors="0">
  <testcase name="verifyOnClickPrevButton" time="0.208" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="invalidWebsitentry" time="11.354" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="verifyNextButtonText" time="0.028" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="isNextButtonVisible" time="0.013" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="hobbiesOptionsVisible" time="0.023" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="hobbiesOptionsVisible" time="0.017" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="isHobbiesLabelVisible" time="0.016" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="isHobbiesCheckBoxVisible" time="0.023" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="selectPicture" time="7.183" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="invalidWebsitentry" time="11.227" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="hobbiesOptionsVisible" time="0.021" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="validWebsiteEntry" time="0.169" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="verifyPrevButtonText" time="0.050" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="isWebsiteTextBoxVisible" time="0.015" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="hobbiesOptionsVisible" time="0.019" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="invalidWebsitentry" time="11.290" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="validHobbiesSelection" time="0.117" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="validWebsiteEntry" time="0.227" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="invalidWebsitentry" time="11.329" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="isPictureLabelVisible" time="0.015" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="invalidWebsitentry" time="11.347" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="validWebsiteEntry" time="0.156" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="hobbiesOptionsVisible" time="0.023" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="isWebsiteLabelVisible" time="0.015" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="isPrevButtonVisible" time="0.015" classname="testScripts.Motorcycle.EnterInsuranceData"/>
=======
<testsuite hostname="a" name="testScripts.Motorcycle.EnterInsuranceData" tests="15" failures="1" timestamp="21 Aug 2020 09:48:09 GMT" time="10.988" errors="5">
  <testcase name="selectPicture" time="10.049" classname="testScripts.Motorcycle.EnterInsuranceData">
    <error type="java.lang.StringIndexOutOfBoundsException" message="String index out of range: -1">
      <![CDATA[java.lang.StringIndexOutOfBoundsException: String index out of range: -1
	at java.lang.String.substring(Unknown Source)
	at pages.Motorcycle.EnterInsuranceData.validPictureSelection(EnterInsuranceData.java:149)
	at testScripts.Motorcycle.EnterInsuranceData.selectPicture(EnterInsuranceData.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
    </error>
  </testcase> <!-- selectPicture -->
  <testcase name="isHobbiesLabelVisible" time="0.041" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="isNextButtonVisible" time="0.035" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="validHobbiesSelection" time="0.247" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="hobbiesOptionsVisible" time="0.000" classname="testScripts.Motorcycle.EnterInsuranceData">
    <error type="org.testng.TestNGException" message="
Data Provider public java.lang.String[] testScripts.Motorcycle.EnterInsuranceData.hobbiesOptions() throws java.io.IOException must return either Object[][] or Iterator&lt;Object&gt;[], not class [Ljava.lang.String;">
      <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.String[] testScripts.Motorcycle.EnterInsuranceData.hobbiesOptions() throws java.io.IOException must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.String;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:157)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1243)
	at org.testng.internal.Invoker.createParameters(Invoker.java:992)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1082)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
    </error>
  </testcase> <!-- hobbiesOptionsVisible -->
  <testcase name="isWebsiteLabelVisible" time="0.031" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="verifyNextButtonText" time="0.074" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="isPrevButtonVisible" time="0.026" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="isPictureLabelVisible" time="0.036" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="verifyOnClickPrevButton" time="0.210" classname="testScripts.Motorcycle.EnterInsuranceData">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at testScripts.Motorcycle.EnterInsuranceData.verifyOnClickPrevButton(EnterInsuranceData.java:97)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
    </error>
  </testcase> <!-- verifyOnClickPrevButton -->
  <testcase name="verifyPrevButtonText" time="0.099" classname="testScripts.Motorcycle.EnterInsuranceData">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at testScripts.Motorcycle.EnterInsuranceData.verifyPrevButtonText(EnterInsuranceData.java:91)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
    </failure>
  </testcase> <!-- verifyPrevButtonText -->
  <testcase name="isHobbiesCheckBoxVisible" time="0.109" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="isWebsiteTextBoxVisible" time="0.031" classname="testScripts.Motorcycle.EnterInsuranceData"/>
  <testcase name="validWebsiteEntry" time="0.000" classname="testScripts.Motorcycle.EnterInsuranceData">
    <error type="org.testng.TestNGException" message="
Data Provider public java.lang.String[] testScripts.Motorcycle.EnterInsuranceData.validWebsiteTestData() throws java.io.IOException must return either Object[][] or Iterator&lt;Object&gt;[], not class [Ljava.lang.String;">
      <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.String[] testScripts.Motorcycle.EnterInsuranceData.validWebsiteTestData() throws java.io.IOException must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.String;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:157)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1243)
	at org.testng.internal.Invoker.createParameters(Invoker.java:992)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1082)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
    </error>
  </testcase> <!-- validWebsiteEntry -->
  <testcase name="invalidWebsitentry" time="0.000" classname="testScripts.Motorcycle.EnterInsuranceData">
    <error type="org.testng.TestNGException" message="
Data Provider public java.lang.String[] testScripts.Motorcycle.EnterInsuranceData.invalidWebsiteTestData() throws java.io.IOException must return either Object[][] or Iterator&lt;Object&gt;[], not class [Ljava.lang.String;">
      <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.String[] testScripts.Motorcycle.EnterInsuranceData.invalidWebsiteTestData() throws java.io.IOException must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.String;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:157)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1243)
	at org.testng.internal.Invoker.createParameters(Invoker.java:992)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1082)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
    </error>
  </testcase> <!-- invalidWebsitentry -->
>>>>>>> af3df79d5ca505e6819af0d6e32521094b9e329f
</testsuite> <!-- testScripts.Motorcycle.EnterInsuranceData -->
